### PR TITLE
Add ports non-oss non-oss/sources for TW

### PR DESCRIPTION
**Pull request by lkocman**
n/a for Leap

### CODE REVIEW

File: opensuse-tumbleweed-ports-repoindex.xml [hunk 1]
```diff
@@ -22,6 +22,18 @@
...
     autorefresh="true"/>
 
+<repo url="%{disturl}/ports/$DIST_ARCH/%{distsub}/repo/non-oss"
+    alias="repo-non-oss"
+    name="%{alias}"
+    enabled="false"
+    autorefresh="true"/>
+
+<repo url="%{disturl}/ports/$DIST_ARCH/source/%{distsub}/repo/non-oss"
+    alias="repo-non-oss"
```
Comment (ggardet) [inline opensuse-tumbleweed-ports-repoindex.xml:32]: Alias should be `repo-non-oss-source` but https://download.opensuse.org/ports/aarch64/source/tumbleweed/repo/non-oss/ does not exists.
(But it exists for `ppc` and `zsystems`.)
Also, not present on `x86_64` XML file.

**Review by ggardet — CHANGES_REQUESTED** [review]

File: opensuse-tumbleweed-ports-repoindex.xml (hunk 1 above)
Comment (lkocman) [reply opensuse-tumbleweed-ports-repoindex.xml:32]: Ah yes nice catch!

Comment (lkocman) [reply opensuse-tumbleweed-ports-repoindex.xml]: https://bugzilla.opensuse.org/show_bug.cgi?id=1226763

**Comment by ggardet** [comment]
Could you squash your commits, please? Otherwise, LGTM.

**Comment by lkocman** [comment]
squashmerged ;-) Thank you!

**Pull request merged**